cosmos/testing/integration/precompile/staking/staking_test.go

```diff
@@ -37,24 +37,24 @@ import (
 	. "pkg.berachain.dev/polaris/cosmos/testing/integration/utils"
 )
 
-func TestCosmosPrecompiles(t *testing.T) {
+func TestStakingPrecompile(t *testing.T) {
 	RegisterFailHandler(Fail)
-	RunSpecs(t, "cosmos/testing/precompile:integration")
+	RunSpecs(t, "cosmos/testing/integration/precompile/staking")
 }
 
 var (
-	tf                *integration.TestFixture
+	stf               *integration.TestFixture
```

Comment:
why stf?

Reply:
yea now that its in different packages, each one can have `tf`